pal03377/Athena-New
# Add support for manual evaluation

### Motivation and Context



Reworked manual ratings from #84 for adding manual evaluation to the playground

### Description


- Add manual ratings to the evaluation mode (accept/reject buttons)
- Add export of manual ratings separately from the results (same button but two files, to make it easier for multiple annotators)
- Add import for manual ratings which can be imported after the results (same button, and we automatically order multiple input files)

### Steps for Testing


1. Run eval mode for a text module
2. Annotate some suggestions and press export, it should export two files (results and manualRatings)
3. Cancel experiment (in module configuration and define the module configuration again)
4. Try out a combination of imports, i.e. first results then manual ratings, both at once -> success | no results then manual ratings or wrong manual ratings (of a different run for example) -> error alert 
5. Repeat for a programming module

### Screenshots


#### Inline Feedback for Text Modules
<img width="608" alt="image" src="https://github.com/ls1intum/Athena/assets/5898705/cf6e4ec7-f5cf-4ca9-9058-8cb06a0f6614">

#### Inline Feedback for Programming Modules
<img width="502" alt="image" src="https://github.com/ls1intum/Athena/assets/5898705/224872b6-a437-4230-87ec-ee1cb89c422a">
